src/app/Header/components/MenuRight/UserAuthModals/ForgetPwd/ForgetPwd.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from 'styled-components';
+import Button from '../../../../../../components/Button';
+import Modal from '../../../../../../components/Modal';
+import config from './formConfig';
+import FormItem from '../../../../../../components/FormItem';
+import Input from '../../../../../../components/Input';
+import useForm from '../../../../../../hooks/useForm';
+
+const Form = styled.form`
+  margin: 0 auto;
+`;
+
+const ConfirmButton = styled(Button)`
+  display: block;
+  margin: 50px auto 28px;
+`;
+
+export default function ForgetPwd({ onClose }) {
+  const {
+    validate, handleChange, values, touched, toggleTouched,
+  } = useForm(
+    config,
+  );
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (!validate()) {
+      toggleTouched();
+    }
```

Comment:
@hachilam92 这是 ziwei 给大家留的坑，这里不需要 if else
```suggestion
    toggleTouched();
```